src/enzo/Grid_FindNewStarParticles.C

```diff
@@ -65,8 +65,10 @@ int grid::FindNewStarParticles(int level)
 	   calls to the IMF. */
 
 	if (ParticleType[i] == -PARTICLE_TYPE_SINGLE_STAR)
-	  if (PopIIIInitialMassFunction == FALSE)
+	  if (PopIIIInitialMassFunction == FALSE){
```

Comment:
Very minor point, but it would be nice if you could add all the brackets (i.e. {}) to this set of if statements (and clean up indentation).